Scripts/script-Round.yml

```diff
@@ -0,0 +1,17 @@
+commonfields:
+  id: Round
+  version: -1
+name: Round
+script: demisto.results(int(round(float(demisto.args()["value"]))))
```

Comment:
you better set the result for some output, instead of just return the result to warroom.
And add some description whether the round will be always rounded down 3.2 => 3 and 3.9 => 3?

Reply:
@anara123 it is a transformer, there is no outputs and it doesn't go to the war room
@asafshen already added floor and ceil as well

Reply:
then why we need round if we have ceil and floor?
And those transformers already available at 3.6?

Reply:
Round is very important, their are cases that you want to round to the closet number, ceil and floor will not help